README.md

```diff
@@ -2,173 +2,159 @@
-         * [Deploy to OpenShift 4.x](#deploy-to-openshift-4x)
-            * [Optional Step](#optional-step)
-            * [Via OperatorHub automatically](#via-operatorhub-automatically)
-            * [Via OperatorHub manually](#via-operatorhub-manually)
-            * [Via local Operator](#via-local-operator)
-            * [Check Operator's availability](#check-operators-availability)
-         * [Deploy to OpenShift 3.11](#deploy-to-openshift-311)
-      * [Trigger a Kogito Runtime Service deployment](#trigger-a-kogito-runtime-service-deployment)
-         * [Deploy a new service](#deploy-a-new-service)
-         * [Clean up a Kogito Service deployment](#clean-up-a-kogito-service-deployment)
-         * [Native X JVM Builds](#native-x-jvm-builds)
-         * [Troubleshooting](#troubleshooting)
-            * [I'm not seeing any builds running!](#im-not-seeing-any-builds-running)
-            * [Native build fails with "The build pod was killed due to an out of memory condition."](#native-build-fails-with-the-build-pod-was-killed-due-to-an-out-of-memory-condition)
-      * [Deploy Data Index Service](#deploy-data-index-service)
-         * [Deploy Infinispan](#deploy-infinispan)
-         * [Deploy Strimzi](#deploy-strimzi)
-            * [Create Required Topics](#create-required-topics)
-         * [Install Data Index Service](#install-data-index-service)
-            * [Retrieve Kafka internal url](#retrieve-kafka-internal-url)
-            * [Install Data Index Service with Kogito CLI](#install-data-index-service-with-kogito-cli)
-            * [Install Data Index Service with Operator Catalog (OLM)](#install-data-index-service-with-operator-catalog-olm)
-            * [Install Data Index Service with oc client](#install-data-index-service-with-oc-client)
+      * [Kogito Operator requirements](#kogito-operator-requirements)
+      * [Kogito Operator installation](#kogito-operator-installation)
+         * [Deploying to OpenShift 4.x](#deploying-to-openshift-4x)
+            * [Automatically in OperatorHub](#automatically-in-operatorhub)
+            * [Manually in OperatorHub](#manually-in-operatorhub)
+            * [Locally on your system](#locally-on-your-system)
+         * [Deploying to OpenShift 3.11](#deploying-to-openshift-311)
+      * [Kogito Runtimes service deployment](#kogito-runtimes-service-deployment)
+         * [Deploying a new service](#deploying-a-new-service)
+         * [Cleaning up a Kogito service deployment](#cleaning-up-a-kogito-service-deployment)
+         * [Native X JVM builds](#native-x-jvm-builds)
+         * [Troubleshooting Kogito Runtimes service deployment](#troubleshooting-kogito-runtimes-service-deployment)
+            * [No builds are running](#no-builds-are-running)
+            * [Native build fails with "The build pod was killed due to an out of memory condition." error](#native-build-fails-with-the-build-pod-was-killed-due-to-an-out-of-memory-condition-error)
```

Comment:
Don't forget also to update the TOC :)

Reply:
Oops! Sorry. Good catch. Fixed. Thank you!